torchrl/envs/libs/vmas.py

```diff
@@ -419,14 +590,10 @@ def read_reward(self, rewards):
         rewards = _selective_unsqueeze(rewards, batch_size=self.batch_size)
         return rewards
 
-    def read_action(self, action):
+    def read_action(self, action, group: str):
```

Comment:
this is a public method and we're adding a new arg without default value, hence it's bc-breaking. Any chance we can make this non-bc breaking?

Reply:
I can, or I can make it private, as requested above, which will be more bc breaking though